options/locale/locale_ko-KR.ini

```diff
@@ -14,18 +14,18 @@ page=페이지
 template=템플릿
 language=언어
 notifications=알림
-create_new=ㅊㄱㄷㅁㅅㄷ...
-user_profile_and_more=ㅖ개랴ㅣㄷ 뭉 ㄴㄷㅅ샤ㅜㅎㄴ...
+create_new=생성하기
+user_profile_and_more=프로파일 및 설정
 signed_in_as=다음 사용자로 로그인됨
-enable_javascript=쏘ㅑㄴ ㅈ듀냣ㄷ 재간 ㅠㄷㅅㅅㄷㄱ 쟈소 ㅓㅁㅍㅁㄴㅊ갸ㅔㅅ.
+enable_javascript=이 웹사이트는 자바스크립트 활성화가 필요합니다.
 
 username=사용자명
 email=뜨먀ㅣ ㅁㅇㅇㄱㄷㄴㄴ
```

Comment:
This string still broken

Reply:
Then we have to wait for anothwr crowdin update

Reply:
@bagasme updated